DataFormats/Phase2TrackerDigi/interface/Hit.h

```diff
@@ -0,0 +1,21 @@
+#ifndef HIT_H
```

Comment:
inclusion guard non conform to rule 4.1 http://cms-sw.github.io/cms_coding_rules.html#4--technical-coding-rules-1